_data/simple-icons.json

```diff
@@ -4379,7 +4379,7 @@
         },
         {
             "title": "Fluxus",
-            "hex": "FFFFFF",
+            "hex": "000000",
```

Comment:
Looking at https://fluxteam.net/, `#000000` does not really represents the brand. I would go for something more purpleish like `#8D73B0` or one of the dark blue value found as background.

Reply:
I prefer to understand that this icon has no theme color. And the `#000000` would be better. The more default scene to use our icons is a white background, if the icon is white there will be visual problems.

Reply:
The color is there to represent the brand color. Not necessarily the icon color (even if it often correspond). In this case, the brand seems to use some color scheme, but never uses black.
We can always ask other @simple-icons/maintainers to weight in on this one!

Reply:
I'm in for the purple!